src/olympia/addons/tests/test_views.py

```diff
@@ -4182,7 +4211,11 @@ def test_enable_disabled_version(self):
         self.version.file.reload()
         assert not self.version.is_user_disabled
         alog = ActivityLog.objects.exclude(
-            action__in=(amo.LOG.LOG_IN.id, amo.LOG.LOG_IN_API_TOKEN.id)
+            action__in=(
+                amo.LOG.LOG_IN.id,
+                amo.LOG.LOG_IN_API_TOKEN.id,
+                amo.LOG.CHANGE_STATUS.id,
```

Comment:
Previously the status change activity caused by the version creation was "hidden" because the task user didn't exist. I needed the task user to be present for the needs human review activity to be logged, so suddenly this caused the status change activity to be recorded...